src/core/Akka/Actor/TypedActor.cs

```diff
@@ -34,7 +35,10 @@ public abstract class TypedActor : ActorBase
         /// <returns>TBD</returns>
         protected sealed override bool Receive(object message)
         {
-            MethodInfo method = GetType().GetMethod("Handle", new[] {message.GetType()});
+            Type[] msgTypeArray = new[] {message.GetType()};
+            Type actorType = GetType();
+            MethodInfo method = actorType.GetMethod("Handle", msgTypeArray) ??
+                                actorType.GetMethod($"Akka.Actor.IHandle<{msgTypeArray[0].FullName}>.Handle", BindingFlags.NonPublic | BindingFlags.Instance);
```

Comment:
This depends on the compiler naming explicit interface implementations like this, which is an implementation detail, and for example the F# compiler uses a different naming scheme afaik.